assets/css/custom.css

```diff
@@ -1126,3 +1126,13 @@ h2.gray-underline:after {
     -webkit-align-items: center;
     color: #fff;
 }
+
+.row.team div.col-md-2 {
```

Comment:
Any difference defining a class like:
```
.fixed-image-size {
...
}
```
and using it in the respective places in the HTML rather how it's done here ?